src/public/style.css

```diff
@@ -171,12 +171,27 @@ body{
     padding: 1rem;
     border-radius: 5px;
     cursor: pointer;
+    transition: all .3s ease;
+}
+
+/* Hover effects */
+.favorite:hover, .pendrive:hover {
+    background: rgba(57, 73, 106, 0.685) !important;
```

Comment:
Nice idea, @Lilkedus! I've tested it and looks very nice on the `dark` theme! However,  I think we need to change the background on hover dynamically instead of a static background value, because we got more than one theme, and Xplorer will support theme extension too! 

The way to do this dynamically is to use javascript, which takes the value of color the user stored on the `theme.json` config file and changes it programmatically.

I will add more commits to this PR before merging it! Anyway, thank you very much!

Regards, Justin

Reply:
Okay, I didn't think about that. I think the javascript implementation would be fairly simple because all I did was change the opacity of the background.